BaseFiles/Common/Common.csproj

```diff
@@ -3400,39 +3430,8 @@
   <ItemGroup>
     <Compile Include="dummy.cs" />
   </ItemGroup>
-  <ItemGroup />
   <ItemGroup>
-    <Folder Include="html\pages\control\widgets\homegenie\generic\locales\" />
-    <Folder Include="html\js\i18n\" />
-    <Folder Include="html\pages\configure\widgeteditor\" />
-    <Folder Include="html\js\codemirror-4.03\mode\css\" />
-    <Folder Include="html\js\codemirror-4.03\mode\htmlembedded\" />
-    <Folder Include="html\js\codemirror-4.03\mode\htmlmixed\" />
-    <Folder Include="html\js\codemirror-4.03\mode\xml\" />
-    <Folder Include="html\pages\configure\interfaces\configlet\" />
-    <Folder Include="html\ui\" />
-    <Folder Include="html\ui\widgets\" />
-    <Folder Include="html\ui\core\" />
-    <Folder Include="lib\" />
-    <Folder Include="html\images\icons\" />
-    <Folder Include="html\images\wallpapers\" />
     <Folder Include="lib\ape\" />
-    <Folder Include="lib\mig\" />
-    <Folder Include="lib\shared\" />
-    <Folder Include="lib\shared\de\" />
-    <Folder Include="lib\shared\es\" />
-    <Folder Include="lib\shared\fr\" />
-    <Folder Include="lib\shared\it\" />
-    <Folder Include="lib\shared\nl\" />
-    <Folder Include="lib\shared\no\" />
-    <Folder Include="lib\shared\ru\" />
-    <Folder Include="lib\shared\tr\" />
-    <Folder Include="lib\shared\uk\" />
-    <Folder Include="lib\shared\zh-CHS\" />
-    <Folder Include="html\pages\control\widgets\weather\wunderground\images\" />
-    <Folder Include="html\css\font-awesome\" />
-    <Folder Include="html\js\app\" />
-    <Folder Include="html\pages\configure\scheduler\templates\" />
-    <Folder Include="html\css\fonts\" />
```

Comment:
Yes, that is wrong. Probably tried some things trying to learn github, HG and something got into this PR that shouldn't be there. Will try to update the PR when I get back from vacation. Only the missing icons should be added in this PR. Sorry for the confusion.